### PR TITLE
Seeker integration

### PR DESCRIPTION
**Pull request by papanikos**
Hi team,

I saw [seeker](https://github.com/gussow/seeker) was on your wishlist #1 #75 , it came up on my and figured I 'd give it a go.

### A short summary of this PR:

* Dockerfile for seeker
  - I am using a conda env within docker. I was testing this and it seems to work
  - I am not sure how you want to handle containers. It would be nice to have them under one "organization". For now I put in my own Docker hub. Feel free to grab it, rewrite it, whatever.
* Seeker run module, filtering based on author's 0.5 threshold, seeker_wf within main phage.nf
* Inclusion in README, minor stylistic changes in there.
* Addition of reference in the the Citations.txt
* Removal of **D**ocs folder, in favor of **d**ocs (maybe a typo in some PR?)

### Profiles tested:
* `-profile smalltest,local,docker`
  - `--anno`, turn off all tools **except** deepvirfinder and seeker
* `-profile smalltest,local,singularity`
  - `--anno`, turn off all tools **except** deepvirfinder and seeker
* `-profile smalltest,slurm,singularity` (our hpc doesn't support docker)
  - full run with all tools, identification and annotation on

All seem to run as expected for me.

### Notes
* I haven't really tested performance of seeker on a big dataset. It provides a minimal command line interface without any options for controlling runtime parameters (e.g. cpu usage). Maybe a wrapper as described [here](https://github.com/gussow/seeker#python-library) included as
 ```bash
run_seeker.py -i <fasta> -o <output.tsv> -c <cores
``` 
in the container would add some flexibility. With that said, in the containers config I used parameters similar to other tools that seemed sensible.

Glad to pick any issue during review. 

Cheers,
Nikos

### CODE REVIEW

**Comment by replikation** [comment]
* thank you very much for this :) , we will review all the steps, test on big datasets and implement this - that's quite helpful
* you installed docker via conda? can you share the code for this, we currently are writing a "read the docs" and could add this part for conda users :)
* good catch on the `--anno` part we might have a bug here on our end

**Comment by papanikos** [comment]
Glad to contribute!

What I meant with "conda env within docker", should be "install conda inside docker (available from the base image anyway) and create an environment inside the docker container"

Some references
* https://pythonspeed.com/articles/activate-conda-dockerfile/
* https://stackoverflow.com/questions/55123637/activate-conda-environment-in-docker

FWIW, I didn't bother with testing if there is any working package available through conda. I had to go through setting up docker on my own Ubuntu 18 laptop.

 Singularity and nextflow work though.
```
$ conda create -n wtp nextflow==20.07.01 singularity==3.6
$ conda activate wtp
(wtp) $ nextflow run replikation/What_the_Phage -r 0.9.0 --setup ...
```
is my current setup, and it works for local profiles.

**Comment by replikation** [comment]
* ah yes i understand now. yep conda install on singularity works, there might be some issues on a cluster environment however compared to a native install of singularity
thanks

File: Docs/Citations.txt [hunk 1]
```diff
@@ -1,193 +0,0 @@
-@article{marquet2020phage,
```
Comment (replikation) [inline Docs/Citations.txt:1]: @mult1fractal this file is removed in this PR
do manual merge so its not removed in the merge

Comment (replikation) [reply Docs/Citations.txt:1]: nvm its renamed to .bib my bad

**Review by replikation — APPROVED** [review]
* @mult1fractal looks good on my end

**Pull request merged**